src/main/java/com/assignment/Assignment8.java

```diff
@@ -0,0 +1,13 @@
+package com.assignment;
+
+public class Assignment8 {
+
+    public int getMiddleNumber(int min, int max) {
+        for (int index = min; index <= max; index++) {
+            if (index == (max + min) / 2) {
```

Comment:
(max + min) / 2 = this will be same till return statement, so define it below getMiddleNumber method, and use it in if, bcoz you are doing (max + min) / 2  this operation multiple time and getting same value